src/main/kotlin/uk/gov/dluhc/printapi/dto/PdfFile.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dluhc.printapi.dto
+
+data class PdfFile(
+    val filename: String,
+    val contents: ByteArray
+) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
Does this need its own equals and hashcode because of the ByteArray field?
A standard data class impl of these methods wont cut it?

Reply:
Correct - this was main reason I hadn't used a `data` class.  But if we are using `data` I guess should follow the guidance for handling the byte array properly 👍